lib/internal/Magento/Framework/ObjectManager/InterceptableValidator.php

```diff
@@ -28,6 +27,20 @@ private function isInterceptor($instanceName)
         return $this->endsWith($instanceName, '\Interceptor');
     }
 
+    /**
+     * Check if instance type is proxy
+     *
+     * @param string $instanceName
+     * @return bool
+     */
+    private function isProxy($instanceName)
```

Comment:
Please add argument & return types to the method.
```suggestion
    private function isProxy(string $instanceName): bool
```

Also would be good to do the same for `isInterceptor` and `isInterceptable` methods

Reply:
Sure, I'll change it.